### ISSUE TITLE
Model training does not improve accuracy

### ISSUE THREAD

**Issue by ChosunOne** [tensorflow/tensorflow#32899]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 19.04
- TensorFlow installed from (source or binary): pip
- TensorFlow version (use command below): 2.0.0-rc2
- Python version: 3.7.3
- CUDA/cuDNN version: release 10.0, V10.0.130
- GPU model and memory: nVidia GTX 1080 Ti

**Describe the current behavior**
When attempting to train a sequential model on the MNIST dataset, the model remains at 11% accuracy.  This is only resolved when the inputs are scaled by 255.

**Describe the expected behavior**
The model should improve in accuracy.

**Code to reproduce the issue**
https://github.com/PacktPublishing/What-s-New-in-TensorFlow-2.0/blob/master/Chapter02/end_to_end_sequential.py

**Other info / logs**
Please reference the issue at https://github.com/PacktPublishing/What-s-New-in-TensorFlow-2.0/issues/1 for more information.  This behavior was not observed in 2.0.0-beta0

**Comment by ravikyram** [comment]
I have tried on colab with TF version 2.0 beta 1, 2.0.0-rc2 and was able to reproduce the issue. Please, find the gist [here.](https://colab.sandbox.google.com/gist/ravikyram/2d329adc843eeca37e3c219e74515b9b/untitled233.ipynb) Thanks!

**Comment by sghoshjr** [comment]
The same Model on Pytorch gives the similar accuracy of around 11%. So I don't suppose its a tf issue.
Anyway, using Adam optimizer or SGD with momentum boosts up the accuracy on both tf and PyTorch.

**Comment by jvishnuvardhan** [comment]
@ChosunOne I agree with @sghoshjr. If you change the learning rate from default `0.01` to `0.0005` gives you ~60% training accuracy. So you need to change

```
model.compile(loss=keras.losses.categorical_crossentropy,
              optimizer=keras.optimizers.SGD(learning_rate=0.0005),
              metrics=['accuracy'])
```

In general, it is always suggested to normalize the inputs data so that you have faster convergence. Even without normalizing input data, you can still get better performance by tuning some hyper parameters. Please check very similar code on TF [website](https://colab.sandbox.google.com/github/tensorflow/docs/blob/r2.0rc/site/en/r2/tutorials/quickstart/beginner.ipynb). If you want better performance, then change learning rate, optimization, normalize data etc.

I am closing the issue as it is resolved. But, please let me know if I'm mistaken.Thanks!
